web/customize_viewer.css

```diff
@@ -2,31 +2,160 @@
  * Custom css for our fork of pdf.js.
  */
 
+html body.light {
```

Comment:
(I'm not too good at intuiting precedences with complex css strings so you'd need to see if that works)